controller/pods.go

```diff
@@ -51,6 +53,113 @@ const (
 	retryDelay = 1 * time.Second
 )
 
+type OpType int
+
+const (
+    Add OpType = iota
+    Delete
+)
+
+// Event struct
+type Event struct {
+	opType OpType
+	cniParams *cni.Parameters
+}
+
+// eventQueue to publish events to
```

Comment:
First letter not capitalized for the go doc and the sentence is cut, 
`// EventQueue to publish events to an event queue instance`